GliaWidgets/Theme/Alert/ConfirmationAlertConfiguration.swift

```diff
@@ -1,8 +1,20 @@
+/// Configuration of a confirmation alert.
 public struct ConfirmationAlertConfiguration {
+    /// Title of the alert.
     public var title: String?
+
+    /// Message of the alert.
     public var message: String?
+
+    /// Title of a negative action button.
```

Comment:
```suggestion
    /// Title of the negative action button.
```

Reply:
Same as above. Changed, but not sure it's needed.